packages/celotool/src/lib/blockscout.ts

```diff
@@ -90,6 +90,7 @@ async function helmParameters(
       envVar.BLOCKSCOUT_SUBNETWORK_NAME,
       celoEnv
     )}"`,
+    `--set blockscout.discord_webhook_url=${fetchEnvOrFallback(envVar.METADATA_CRAWLER_DISCORD_WEBHOOK, '')}`,
```

Comment:
I know we still have this on the blockscout helm chart, but maybe separate it to something like metadata-crawler namespace?